app/src/main/java/com/example/myapplication/ui/MainActivityComposable.kt

```diff
@@ -140,37 +141,46 @@ fun ProductInfo(modifier: Modifier = Modifier, product: Product, onAttributeClic
                 modifier = Modifier.padding(0.dp, vertical = 8.dp)
             )
             Text(
-                text = stringResource(R.string.description),
+                text = NNSettingsString("BrandDescription", stringResource(R.string.description)),
```

Comment:
Just a note :) You don't need a default value, but I like it :) Anyways you can merge this nice job :)